relnotes/v0.38.0.md

```diff
@@ -10,7 +10,7 @@
 
 ### Bug fixes
 
-* Add `require 'time'` to `remote_config.rb` to avoid "undifined method `rfc2822'". ([@necojackarc][])
+* Add `require 'time'` to `remote_config.rb` to avoid "undifined method \`rfc2822'". ([@necojackarc][])
```

Comment:
You mean I should replace the closing `'` with another backtick instead? This "open with `` ` ``, close with `'` " style is very common in Ruby errors, so I'd say the original is the correct error message.